tests/es-bulk-retry.sh

```diff
@@ -59,6 +54,12 @@ ruleset(name="try_es") {
 		}
 		# else fall through to retry operation
 	}
+	if strlen($.omes!_id) > 0 then {
+		set $.es_msg_id = $.omes!_id;
+	} else {
+		# NOTE: depends on rsyslog being compiled with --enable-uuid
```

Comment:
I think it would be good to check this inside ./tests/Makefile.am, so that the test is not executed if this is not given.

Reply:
oh wait.. maybe I was wrong in this instance, let me check...